ts_scripts/torchserve_grpc_client.py

```diff
@@ -147,7 +339,9 @@ def unregister(stub, model_name):
     if args.action == "infer":
         infer(get_inference_stub(), args.model_name, args.model_input, metadata)
     elif args.action == "infer_stream":
-        infer_stream(get_inference_stub(), args.model_name, args.model_input, metadata)
+        infer_stream(get_inference_stub(), args.model_name, args.model_input)
+    elif args.action == "infer_stream2":
+        infer_stream2(args.model_name, args.sequence_id, args.input_files)
```

Comment:
Why not put a sequence inside a single file?

Reply:
A single file can contains a sequence of requests. Then, we need define the input file format to present the request (eg. each line is the filename of a request)

I think cx needs both single input file and a list of input files. Here , we assume a list of request file for demo.